.env.example

```diff
@@ -66,3 +66,5 @@ ADALITE_ENABLE_SEARCH_BY_TICKER=true
 # catalyst next round start and end dates in ISO format (example: 2021-2030)
 ADALITE_NEXT_VOTING_START=2021-01-01T00:00:00.000Z
 ADALITE_NEXT_VOTING_END=2030-01-01T00:00:00.000Z
+
+TOKEN_REGISTRY_URL="https://tokens.cardano.org"
```

Comment:
the convention is to namespace "adalite" env variables with the "adalite" prefix to eliminate collisions with unrelated env variables - for this case, it may look a bit awkward but I'd rather stick to our convention

Reply:
fixed